testsuites/syncgateway/functional/tests/test_log_rotation_new.py

```diff
@@ -0,0 +1,609 @@
+import json
+import os
+import pytest
+
+from keywords.ClusterKeywords import ClusterKeywords
+from keywords.remoteexecutor import RemoteExecutor
+from keywords.SyncGateway import SyncGateway, sync_gateway_config_path_for_mode, load_sync_gateway_config, get_sync_gateway_version
+from keywords.utils import log_info, host_for_url
+from libraries.testkit.cluster import Cluster
+from keywords.exceptions import ProvisioningError
+
+
+@pytest.mark.sanity
+@pytest.mark.syncgateway
+@pytest.mark.logging
+@pytest.mark.parametrize("sg_conf_name", ["log_rotation_new"])
+def test_log_rotation_default_values(params_from_base_test_setup, sg_conf_name):
+    """Test to verify default values for rotation section:
+    maxsize = 100 MB
+    MaxAge = 0(do not limit the number of MaxAge)
+    MaxBackups = 0(do not limit the number of backups)
+    """
+    cluster_conf = params_from_base_test_setup["cluster_config"]
+    mode = params_from_base_test_setup["mode"]
+    sg_conf = sync_gateway_config_path_for_mode(sg_conf_name, mode)
+    cluster_helper = ClusterKeywords()
+    cluster_hosts = cluster_helper.get_cluster_topology(cluster_conf)
+    sg_admin_url = cluster_hosts["sync_gateways"][0]["admin"]
+    sg_ip = host_for_url(sg_admin_url)
+
+    log_info("Using cluster_conf: {}".format(cluster_conf))
+    log_info("Using sg_conf: {}".format(sg_conf))
+
+    if get_sync_gateway_version(sg_ip)[0] < "2.1":
+        pytest.skip("Test NA for SG  < 2.1")
```

Comment:
may be good idea to put a reason , you can put something like "log rotation with this feature not applicable to run with with below 2.1

Reply:
Done